src/main/kotlin/kweb/util/misc.kt

```diff
@@ -30,12 +29,21 @@ val scheduledExecutorService: ScheduledExecutorService = Executors.newScheduledT
 fun String.escapeEcma() = StringEscapeUtils.escapeEcmaScript(this)!!
 
 val gson = Gson()
-//TODO I'm not sure what the best way to deal with this is.
-fun Any.toJson(): String = gson.toJson(this)
+fun ChartConfig.toJson() : String = gson.toJson(this).toString()
+fun String.toJson(): String = Json.encodeToJsonElement(this).toString()
```

Comment:
Where are these extension functions being used? Seems like they should be deleted.

Reply:
These were written to replace Any.toJson() like we discussed. As I change more and more calls to JsonPrimitive instead of toJson(), I see that you are right, and they can be removed.